packages/cli/src/databases/migrations/sqlite/1745587087521-AddWorkflowStatisticsRootCount.ts

```diff
@@ -0,0 +1,5 @@
+import { AddWorkflowStatisticsRootCount1745587087521 as BaseMigration } from '../common/1745587087521-AddWorkflowStatisticsRootCount';
+
+export class AddWorkflowStatisticsRootCount1745587087521 extends BaseMigration {
+	transaction = false as const;
+}
```

Comment:
I saw this a couple of times now. Do we know why we're doing this or do we copy this from old migrations? Because I've never done that and I tested this one without disabling transactions and it worked 🤔

Reply:
to see why this is needed, create a sqlite migration locally that adds or removes a column from the worfklows table. after the migration runs, all your workflows will most likely disappear in the UI.